resources/css/tag.css

```diff
@@ -1,13 +1,38 @@
 .tag {
-  display: inline-block;
-  opacity: .9;
+  display: inline-flex;
+  align-items: center;
+  opacity: 0.9;
   margin: 0 .15em;
-  padding: .1em .4em;
+  padding: 0 .45em;
   border-radius: .2em;
+  background-color: currentColor;
   vertical-align: text-bottom;
   line-height: 1.1em;
   font-size: .8em;
-  background: var(--text-color);
-  border: 1px solid var(--embed-highlight-color);
-  color: var(--embed-color);
+  /* border: 1px solid var(--embed-highlight-color); */
+}
+
+.tag .label {
+  margin-left: -.45em;
+  padding: .15em .3em .15em .45em;
+  border-radius: .2em 0 0 .2em;
+  background-color: rgba(255, 255, 255, 0.5);
+  color: black;
+}
+
+.tag .arrow-right {
+  width: 0;
+  height: 0;
+  border: solid .7em transparent;
+  border-left-color: rgba(255, 255, 255, 0.5);
+  border-right-width: 0;
+}
+
+.tag .inner {
+  padding: .15em 0;
+  color: black;
+}
+
+.tag .label ~ .inner {
+  margin-left: .2em;
```

Comment:
It doesn't need two classes for both label and inner; just a class for the highlighted part (and the arrow). Just adding a span without the class can then inherit default behaviour. 

The color could be theme aware using a css var.

```css
.tag > * {
  color: var(--bg-color);
}

.tag-sub {
  display: inline-block;
  margin: -.1em .4em -.1em -.4em;
  padding: .1em .4em;
  background: rgba(var(--highlight-color-values), 50%);
}
```

`.tag-sub` would be the same as `.label` here - could be `.tag-label` to be more specific in this case.

Reply:
>It doesn't need two classes for both label and inner

I recall I merely created `.inner` because tags in general needed the inner span (for `currentColor` to work) and I didn't want to leave just a generic `<span>$category</span>` there.

Reply:
```css
background: rgba(var(--highlight-color-values), 50%);
```
Wow. Had no idea this was even possible.

Reply:
Ad inner class: I can understand. Yet, I removed so many confusing and unused classes in this project already that I'm careful to add any new 😅 I'm personally fine with a structural element without a class.

Reply:
> ```css
> background: rgba(var(--highlight-color-values), 50%);
> ```

Yeah, that was a new one for me, too. However, didn't end up using it in the other commits :) white with just opacity as you did it works much better.

Reply:
>I'm personally fine with a structural element without a class

I think it's better your way. Sadly webdev-driven OCD leads us to want a class in every `<span>` and `<div>` we can find. 🤓